contracts/mocks/docs/ERC4626Fees.sol

```diff
@@ -64,27 +69,33 @@ abstract contract ERC4626Fees is ERC4626 {
         }
     }
 
-    function _entryFeeBasePoint() internal view virtual returns (uint256) {
-        return 0;
+    // === Fee configuration === ///
+
+    function _entryFeePCM() internal view virtual returns (uint256) {
+        return 0; // i.e. 1_000 for 1%
     }
 
-    function _entryFeeRecipient() internal view virtual returns (address) {
-        return address(0);
+    function _exitFeePCM() internal view virtual returns (uint256) {
+        return 0; // i.e. 1_000 for 1%
     }
 
-    function _exitFeeBasePoint() internal view virtual returns (uint256) {
-        return 0;
+    function _entryFeeRecipient() internal view virtual returns (address) {
+        return address(0); // i.e. treasury
     }
 
     function _exitFeeRecipient() internal view virtual returns (address) {
-        return address(0);
+        return address(0); // i.e. treasury
     }
 
-    function _feeOnRaw(uint256 assets, uint256 feeBasePoint) private pure returns (uint256) {
-        return assets.mulDiv(feeBasePoint, 1e5, Math.Rounding.Ceil);
+    // === Fee operations === ///
+
+    /// @dev Calculates fee on `assets` not including fees. Used in {IERC4626-mint} and {IERC4626-withdraw} operations.
```

Comment:
```suggestion
    /// @dev Calculates fee to apply on an amount `assets` that does already include fees. Used in {IERC4626-mint} and {IERC4626-withdraw} operations.
```

Reply:
I consciously decided to be short here because at this point I'd use 2 lines instead, and because these are the docs I'd prefer it to be the less bloated possible.

Same applies to the other one.

Thoughts @frangio?

Reply:
2 lines is good (we can still use `///`). "fee on `assets` not including fees" sounds very confusing.

I believe there is an error in @Amxx's suggestion, should say "does _not_ already include".